jinahub/encoders/audio/VGGISHAudioEncoder/tests/unit/test_exec.py

```diff
@@ -25,6 +25,7 @@ def test_embedding_dimension():
     ops.reset_default_graph()
     model = VggishAudioEncoder()
     model.encode(doc, parameters={})
+    assert doc[0].embedding.shape[0] == 128
     assert doc[0].embedding.shape[-1] == 128
```

Comment:
so the embedding shape should be `doc[0].embedding.shape == (128,0)`?

Reply:
Yes, I have updated the code to assert the above. Please see [here](https://github.com/jina-ai/executors/pull/31/commits/adc062852447ab0fa23e4b5dbc3cbcdba9e083ff) for the change.